### DISCUSSION TITLE
QOL Thoughts

### DISCUSSION THREAD

**Discussion by ChandlerHunter** [Confuzu/CivitAI_Image_grabber#26] [Ideas]
I've used this once and it was exactly what I needed. The only improvement I suggest is more targeted search( if thats the right way to say that). IF POSSIBLE ( i didnt do research on what the api allows)

For example:  I downloaded images from a user that wanted a lora created of a character they made. However I could not specify tags on the images or the rating. 

Downloading photos from models can be a trip too. Some LORAs have images that dont represent the intention and searching with tags and rating would be nice. (i'm not sure if thats mode 3). 

OH and maybe a user changeable limit. Either pick a number or no limit

So time is saved but sorting through the photos can make it up in some cases. Fortunately I used Dataset Processor and it helped me sort them.

**Comment by Confuzu** [comment]
for the tag search there is option 3 and to get all tags that civit knows the helper script tagnames.py is provided. But I can't guarantee if all the tags the API spits out really work. This is due to CivitAI not the script.
As for your idea with the rating, I can have a look at how this could be implemented.
What  do you mean with  "user changeable limit. "

**Reply by ChandlerHunter** [reply]
Like limit the amount of photos it downloads, instead of everything

**Reply by Confuzu** [reply]
check  the newest update

**Comment by Kucklin** [comment]
Good evening, how to install CivitAI Image grabber, I am a newbie, I would like to download images from Civitai.com

**Reply by Confuzu** [reply]
Hello,  sorry for the late response.
It is very easy, just download the zip https://github.com/Confuzu/CivitAI_Image_grabber/archive/refs/heads/main.zip.
And from there, follow the usage Section of the README.
